process/generic_worker.go

```diff
@@ -38,7 +39,10 @@ func (g *Generic) processNextWorkItem() bool {
 
 	holder := g.Controllers[key.gvk]
 	logger := g.logger.With(logz.NamespaceName(key.Namespace),
-		logz.ControllerName(key.Name), logz.Iteration(atomic.AddUint32(&g.iter, 1)))
+		logz.ControllerGk(key.gvk.GroupKind()),
```

Comment:
We don't need to set both gks every time - IIRC this is done once when the logger is constructed for the `Generic`.

Reply:
See big comment